test/ResultSet/HydratingResultSetIntegrationTest.php

```diff
@@ -29,4 +29,27 @@ public function testCurrentWillReturnBufferedRow()
         $obj2 = $hydratingRs->current();
         $this->assertSame($obj1, $obj2);
     }
+
+    /**
+     * @covers Zend\Db\ResultSet\HydratingResultSet::current
+     */
+    public function testCurrentUsesPrototypeFactory()
+    {
+        $dataSource = new \ArrayIterator([
+            ['id' => 1, 'name' => 'one'],
+            ['id' => 2, 'name' => 'two'],
+        ]);
+
+        $factory = $this->getMock('\Zend\Db\ResultSet\HydratingResultSet\PrototypeFactoryInterface');
```

Comment:
Class names inside strings are always considered globally qualified, and do not need the `\\` prefix. (Also seen in line 53.)